spec/knapsack_pro/runners/queue/rspec_runner_spec.rb

```diff
@@ -274,10 +291,33 @@
           expect(KnapsackPro::Hooks::Queue).to receive(:call_after_queue)
           expect(KnapsackPro::Report).to receive(:save_node_queue_to_api)
 
-          expect(subject).to eq({
-            status: :completed,
-            exitstatus: exitstatus,
-          })
+          expect(logger).to receive(:info)
+            .with('To retry all the tests assigned to this CI node, please run the following command on your machine:')
+          expect(logger).to receive(:info).with(logged_rspec_command_matcher)
+        end
+
+        context 'when @@used_seed has been set' do
+          let(:used_seed) { '8333' }
+          let(:logged_rspec_command_matcher) { /#{args.join(' ')} --seed #{used_seed} \"#{all_test_file_paths.join(' ')}\"/ }
```

Comment:
Sometimes I wonder what to do in a situation like this.
It's easier to use programming in the spec to generate the expected string. I often do it because I like the convenience. But on the other hand, we can make mistakes in the spec as well. Who is going to test the spec itself? - once, a friend asked me :)

I had to put some mental effort into figuring out what we actually expected as a string here. It wasn't clear from reading the code.

I see one potential problem. For now, we assume `let(:all_test_file_paths) { ['a_spec.rb'] }` has one spec only.
If there would be more specs, then the `\"#{all_test_file_paths.join(' ')}\"` code does not generate a string that we actually should expect. Each spec file should be inside of apostrophes. For example, for 2 specs it should be `"a_spec.rb" "b_spec.rb"`. It's needed to ensure RSpec can run spec files from a directory with a space in the name.
Currently your coude would generated incorrect string `"a_spec.rb b_spec.rb"`.

You may consider using the expected string explicitly here so that reading the code is easier.
But I'm leaving it up to you. Whatever you prefer, it's fine with me.

Reply:
I agree. I'm changing this to simply include a literal path in the expectation.